### PR TITLE
 Add a note on automasking

### PR DESCRIPTION
**Pull request by hongliliu**
Note on automasking.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -1 +1,5 @@
 Data reduction repository for ALMA-IMF
+The current line_imaging.py makes use of the auto-masking method by combing both auto-multithresh and multiscale. 
+Nevertheless, this combination has not been extensively tested as noted in CASAguide at 
+https://casaguides.nrao.edu/index.php/M100_Band3_Combine_5.4.
```
Comment (keflavich) [inline README.txt]:
```suggestion
https://casaguides.nrao.edu/index.php/M100_Band3_Combine_5.4.  When constructing your
own parameter sets in `imaging_parameters.py`, please try using `hogbom` clean instead.
We have not yet converged on the best set of parameters for line cleaning.
```

Comment (hongliliu) [reply README.txt]: It is much better than mine.

Comment (keflavich) [reply README.txt]: Could you merge the suggestion, then?

**Comment by keflavich** [comment]
Looks good.  You should have been able to merge my suggestion from within the browser interface, but it looks like you made the changes on your local clone instead.   That’s totally fine, it doens’t cause any problems, but the browser interface is supposed to make your life easier.

**Pull request merged**